qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -182,10 +182,12 @@ def install_deps(self):
         self.mount_a.client_remote.run(args=args, omit_sudo=False)
 
     def create_reqd_users(self):
-        self.mount_a.client_remote.run(args=['sudo', 'useradd', 'fsgqa'],
+        self.mount_a.client_remote.run(args=['sudo', 'useradd', '-m', 'fsgqa'],
                                        omit_sudo=False, check_status=False)
         self.mount_a.client_remote.run(args=['sudo', 'groupadd', 'fsgqa'],
                                        omit_sudo=False, check_status=False)
+        self.mount_a.client_remote.run(args=['sudo', 'useradd', 'fsgqa2'],
```

Comment:
Why is this additional user required?

Reply:
It's included in xfstests-dev's `README`, so I added it to our codebase. When this is not included some test might be skipped. And since we wish to run all tests from xfstests-dev, I thought it's best to add it.

Reply:
What tests are those and how many of them? And do those additional test pass?

Reply:
Since we talked about this on the call, I am marking this conversation resolved.